### ISSUE TITLE
Fireflow Module- No parameter to pass application as input 

### ISSUE THREAD

**Issue by atm2019** [algosec/algosec-python#49]
If we need to create a change request for ICMP rule in Algosec through algosec python module, currently there is no option to  provide application parameter input. It will great if application parameter can be provided with the fireflow.py. It will help to deploy the ICMP rules by creating change request in algosec.

**Comment by AlmogCohen** [comment]
Hi there! I'm sorry to have somehow missed your message somehow. I look deeper into the details and will respond early during the week, hopefully tomorrow!

**Comment by AlmogCohen** [comment]
@atm2019 I'm pretty sure it is currently possible to request ICMP in the Change request creation process. As you can see [here](https://github.com/algosec/algosec-python/blob/master/algosec/api_clients/fire_flow.py#L116), the `create_change_request` method accepts `traffic_lines`. As can be seen [here](https://github.com/algosec/algosec-python/blob/master/algosec/models.py#L179), TrafficLine object does get a list of services, which can be anything AlgoSec understand. I didn't test that specifically but I'm pretty sure you can use this while specifiying ICMP as one of the services of the traffic line.

Please let me know if I made it clear and if it does solve the issue you have been facing. I'm happy to help in any way.

**Comment by atm2019** [comment]
It is possible to pass ICMP through script. But we don’t want to configure
entire ICMP 0-255 on firewalls. We are looking at specific ICMP type to be
configured on firewalls. Like ICMP[0], ICMP[8], ICMP[12] alone to be
configured on firewall. Currently only ICMP[8] can be pushed through script
if we pass the service value as “ping”.

**Comment by AlmogCohen** [comment]
@atm2019 Oh, I understand now... I've checked internally and unfortunately it seems that it is not supported by the API at the moment not by the SDKs...

**Comment by atm2019** [comment]
Noted. When we try to pass the service as “ping” through script , the change request gets stuck in fireflow planning stage and no progress . We are using 2018.2 versio

**Comment by AlmogCohen** [comment]
@atm2019 please let me understand - the CR is successfully created via the SDK and then is stuck somewhere down the processing line inside FireFlow?

**Comment by atm2019** [comment]
Yes

**Comment by AlmogCohen** [comment]
As we have discussed over emails, I've released [v1.3.3](https://github.com/algosec/algosec-python/releases/tag/v1.3.3) following the support added for traffic applications in  https://github.com/algosec/algosec-python/pull/54.

A traffic line in the new format can use the traffic application like that:
```
ping_traffic_line = ChangeRequestTrafficLine(
    ChangeRequestAction.ALLOW,
    ['source1', 'source2'],
    ['dest1', 'dest2'],
    [],
    ['ping'],
)
```

Please let me know if you need any further help!
